fe/fe-core/src/main/java/com/starrocks/persist/AlterMaterializedViewBaseTableInfosLog.java

```diff
@@ -0,0 +1,94 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package com.starrocks.persist;
+
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.catalog.BaseTableInfo;
+import com.starrocks.catalog.MaterializedView;
+import com.starrocks.catalog.MvId;
+import com.starrocks.common.io.Text;
+import com.starrocks.common.io.Writable;
+import com.starrocks.persist.gson.GsonUtils;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+public class AlterMaterializedViewBaseTableInfosLog implements Writable {
+
+    @SerializedName(value = "dbId")
+    private long dbId;
+    @SerializedName(value = "mvId")
+    private long mvId;
+    @SerializedName(value = "baseTableInfos")
+    private List<BaseTableInfo> baseTableInfos;
+    @SerializedName(value = "remoteMvId")
+    private MvId remoteMvId;
+    @SerializedName("baseTableVisibleVersionMap")
+    private Map<Long, Map<String, MaterializedView.BasePartitionInfo>> baseTableVisibleVersionMap;
```

Comment:
why store the versionmap instead of refreshing the mv after restoring ?

Reply:
After restore, we only need to remap the remote db.table to the new db.table by changing the version map instead of refreshing mv, so can use this mv directly.

Because backup/restore has already backup/restore the mv's data totally, we no need to refresh it anymore.

Ofcourse there are a lot of checks to achieve Active state in the `doAfterRestore` method.